CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+gallery (16.1.0)
+* Rotate images in the browser [#764](https://github.com/owncloud/gallery/pull/764)
+* Do not break if mount is not available [#739](https://github.com/owncloud/gallery/pull/739)
+* Fix to work properly with 10.0.9 [738](https://github.com/owncloud/gallery/pull/738)
```

Comment:
did you test 16.1.0 with OC 10.0.8 or do we need min-version=10.0.9 ?

Reply:
yes - tested with 10.0.8